spring-cloud-gcp-samples/spring-cloud-spanner-r2dbc-samples/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <!-- Your own application should inherit from spring-boot-starter-parent -->
+    <artifactId>spring-cloud-gcp-samples</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.5.5-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>spring-cloud-spanner-r2dbc-samples</artifactId>
+  <name>Spring Framework on Google Cloud Code Sample - Spanner-r2dbc</name>
+
+  <properties>
+    <sonar.skip>true</sonar.skip>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+    <dependency>
+        <groupId>org.springframework.data</groupId>
+        <artifactId>spring-data-r2dbc</artifactId>
+        <version>1.5.13</version>
+      </dependency>
+      <dependency>
+        <groupId>io.r2dbc</groupId>
+        <artifactId>r2dbc-spi</artifactId>
+        <version>0.9.0.RELEASE</version>
+      </dependency>
```

Comment:
yes the transitive dependencies are not required to be declared here. I have removed those. 

Also @meltsufin  I noticed that other modules are having a starter's module mentioned in the documentation , so on the similar lines I have added a new module  `spring-cloud-gcp-starter-spanner-r2dbc` under starters and have referred it @ `spanner-spring-data-r2dbc.adoc` . Please let me know if it looks good

Reply:
The starters are there to integrate with auto-configuration. Since we don't have auto-configuration for this module yet, a starter doesn't make sense.
That being said, we should add auto-configuration in the future. Please add it to the list of tasks.

Reply:
sure @meltsufin , I have removed the starter's module and have mentioned `spring-cloud-spanner-spring-data-r2dbc` artifact in the `adoc`